docs/conf.py

```diff
@@ -80,7 +80,7 @@ def __getattr__(cls, name):
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = "en"
+language = None
```

Comment:
please don't modify default config, this is deployment specific and can be overwritten